### PR TITLE
Improve Docker CI for cloud team

### PR DESCRIPTION
**Pull request by curquiza**
Work with @eskombro 
Update CI to send a signal to Cloud team CI when Docker image is pushed

### CODE REVIEW

File: .github/workflows/publish-docker-images.yml [hunk 1]
```diff
@@ -62,10 +62,19 @@ jobs:
...
       - name: Build and push
-        id: docker_build
         uses: docker/build-push-action@v3
         with:
           # We do not push tags for the cron jobs, this is only for test purposes
           push: ${{ github.event_name != 'schedule' }}
           platforms: linux/amd64,linux/arm64
           tags: ${{ steps.meta.outputs.tags }}
+
+      # /!\ Don't touch this without checking with Cloud team
```
Comment (eskombro) [inline .github/workflows/publish-docker-images.yml:72]: <3

**Review by eskombro — COMMENTED** [review]
Work as a charm 👌🏽
Thanks 🍊

**Review by eskombro — APPROVED** [review]
🦭

**Review by eskombro — APPROVED** [review]

**Review by irevoire — APPROVED** [review]

**Comment by curquiza** [comment]
bors merge

**Pull request merged**